src/obstacle_detector.cpp

```diff
@@ -217,6 +289,25 @@ void ObstacleDetector::get_points()
     obstacle_mutex.unlock();
 }
 
+// Get points from lidar - convert from polar to cartesian coords
+void ObstacleDetector::get_lidar_points(std::vector<tf2::Vector3>& points)
```

Comment:
It doesn't 'get', since it doesn't return. Call it 'add_lidar_points'

Reply:
Actually it may be worth keeping it `get` so it is consistent in naming with the range senors